src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
```

Comment:
###AWAMQyizS1b073eKUTZL###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_7@6
Complete the task associated to this TODO comment.

Reply:
simulating reply on outdated comment

Reply:
simulating auto reply on outdated comment